.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
these need to be headers (small, h3 maybe?) or there needs to be two linebreaks afterwards, one linebreak puts them on the same line 
![image](https://user-images.githubusercontent.com/31628143/115128790-470b9780-9fae-11eb-9265-59240edb16cd.png)